code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -34,10 +37,14 @@
 	return ..()
 
 /obj/item/melee/sickly_blade/attack_self(mob/user)
+	if(!seek_safety(user))
+		return ..()
```

Comment:
You made it do a thing when it returns TRUE but you didn't make this proc return TRUE anyways

Reply:
uh i think i did what you wante dme to do